src/engine/strat_engine/thinpool/filesystem.rs

```diff
@@ -220,6 +220,13 @@ impl StratFilesystem {
                 // TODO: do anything when filesystem is not mounted?
                 // TODO: periodically kick off fstrim?
             }
+            ThinStatus::Error => {
+                let error_msg = format!(
+                    "Unable to get status for filesystem thin device {}",
+                    self.thin_dev.device()
+                );
```

Comment:
Would it be useful to include the fs name in this error message?  Also asking for all the other error messages that are similar in PR.

Reply:
Filesystem name is not available here.